.buildkite/pipeline.yml

```
@@ -1,5 +1,118 @@
 # yaml-language-server: $schema=https://raw.githubusercontent.com/buildkite/pipeline-schema/main/schema.json
 
+env:
+  SETUP_GVM_VERSION: "v0.5.1"
+  GOLANG_VERSION: "1.21.3"
+  IMAGE_UBUNTU_X86_64: "family/core-ubuntu-2004"
+  IMAGE_UBUNTU_ARM_64: "core-ubuntu-2004-aarch64"
+  DOCKER_REGISTRY: "docker.elastic.co"
+  STAGING_IMAGE: "${DOCKER_REGISTRY}/observability-ci"
+  DOCKER_REGISTRY_SECRET_PATH: "secret/observability-team/ci/docker-registry/prod"
+  BUILDX: 1
+
 steps:
-  - label: "Example test"
-    command: echo "Hello!"
+  - group: "Staging"
+    key: "staging"
+
+    steps:
+      - label: ":unix: Staging / Ubuntu X86_64 - {{matrix.makefile}}"
```

unix is not part of the emoji list available in Buildkite
https://github.com/buildkite/emojis#emoji-reference

What about using :linux:? 

```suggestion
      - label: ":linux: Staging / Ubuntu X86_64 - {{matrix.makefile}}"
```